build.gradle

```diff
@@ -158,7 +158,7 @@ configure(scalaProjects) {
                     "-feature",
                     "-unchecked",
                     "-language:reflectiveCalls",
-                    "-Wconf:cat=deprecation:ws,any:e",
```

Comment:
Any reason why we're removing the previous flag to report other warnings as error ?

Reply:
For some reason that now turned all warnings into errors - so that caused the compile to fail. Where as in 2.13.6 it didnt.